src/lib/Scenes/Artwork/Components/CommercialButtons/InquirySuccessNotification.tsx

```diff
@@ -0,0 +1,94 @@
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+import { CloseIcon, Flex, Text } from "palette"
+import React, { useEffect, useRef } from "react"
+import { Animated, Modal, TouchableOpacity, TouchableWithoutFeedback } from "react-native"
+import styled from "styled-components/native"
+
+interface InquirySuccessNotificationProps {
+  modalVisible: boolean
+  toggleNotification: (state: boolean) => void
+}
+
+export const InquirySuccessNotification: React.FC<InquirySuccessNotificationProps> = ({
+  modalVisible,
+  toggleNotification,
+}) => {
+  let delayNotification: NodeJS.Timeout | any
+  const navRef = useRef(null)
+  const navigateToConversation = () => {
+    toggleNotification(false)
+    SwitchBoard.presentNavigationViewController(navRef as any, "inbox")
+  }
+
+  useEffect(() => {
+    delayNotification = setTimeout(() => {
+      toggleNotification(false)
+    }, 2000)
+    return () => {
+      clearTimeout(delayNotification)
+    }
+  }, [modalVisible])
+
+  return (
+    <Modal
+      visible={modalVisible}
+      onRequestClose={() => toggleNotification(false)}
+      animationType="fade"
+      transparent
+      ref={navRef}
+    >
+      <Animated.View
+        style={{
+          position: "absolute",
+          left: 0,
+          right: 0,
+          bottom: 0,
+          height: "95%",
```

Comment:
Rather than using a proportional height to avoid the notch here we can use `safeAreaInsets` to make sure this works on all devices.

```suggestion
          top: useScreenDimensions().safeAreaInsets.top
```